File: 02. CONV_UNI_SOAP_DOTNET_GUARDERAS/02. CLIMOV/WS_CONVUNI_SOAPDOTNET_CLIMOV/app/src/main/java/ec/edu/monster/prueba/Prueba01.java
package ec.edu.monster.ws_convuni_soapdotnet_climov.prueba;

import ec.edu.monster.ws_convuni_soapdotnet_climov.controlador.ConvUniControlador;

public class Prueba01 {
    public static void main(String[] args) {
        //Datos
        double c = 0;
        double f = 32;

        //Proceso
        ConvUniControlador servicio = new ConvUniControlador();
        String convF = servicio.convertCF(String.valueOf(c));
        String convC = servicio.convertFC(String.valueOf(f));

        //Reporte
        System.out.println(c + "°C = " + convF + "°F");
        System.out.println(f + "°F = " + convC + "°C");
    }
}

File: 02. CONV_UNI_SOAP_DOTNET_GUARDERAS/02. CLIESC/CLIESC/CLIESC/ec.edu.monster.prueba/Prueba01.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CLIESC.ec.edu.monster.controlador;

namespace CLIESC.ec.edu.monster.prueba
{
    public class Prueba01
    {

        public static void Main(string[] args)
        {
            double c = 0;
            double f = 32;

            ConvUniControlador controlador = new ConvUniControlador();
            object nuevF = controlador.convertCF(c);
            object nuevC = controlador.convertFC(f);

            Console.WriteLine(c + "°C = " + nuevF + "°F");
            Console.WriteLine(f + "°F = " + nuevC + "°C");
        }
    }
}
